docs/using.rst

```diff
@@ -8,9 +8,11 @@ Principles
 * Consider the ADB to be ephemeral and do not store data in it long term. It is
   recommended that you do the following:
 
-  * back up your code
-  * use a source control system
-  * mount your code into the box from your host system
+  * Back up your code.
+  * Use a source control system.
+  * Mount your code into the box from your host system.
+    
+    * In case you are looking for bi-directional (i.e. 2 way) folder sync please Refer the `relevant section for bi-directional folder sync <#vagrant-bi-directional-folder-sync>`_.
```

Comment:
ok, at least we use 'bi-driectional' now, but what do you mean with (i.e. 2 way)? That is a superfluous statement at this point. The very definition of bi-directional is that works in both directions